packages/react-scripts/template/src/components/example/WagonWheelControl.js

```diff
@@ -69,7 +69,7 @@ const WagonWheelControl = ({ animationDuration, color, handleColorChange, handle
             colors.feedback.warning.accent,
           ].map((c) => {
             return (
-              <Cell key={c} columns={atSize({ xs: 2, sm: 1 })}>
+              <Cell key={c} columns={atSize({ xs: 2, sm: 2, md: 2, lg: 1 })}>
```

Comment:
there are new values for this now in 8, at least the docs said so

Reply:
On it.